### PR TITLE
Add support for config flag

### PR DESCRIPTION
**Pull request by l4u**
related #196

### CODE REVIEW

**Comment by ryfloyd** [comment]
@l4u Won't this template `initViper` function need to change as well? https://github.com/volatiletech/sqlboiler/blob/master/templates_test/singleton/boil_main_test.tpl#L61

File: main.go [hunk 1]
```diff
@@ -17,45 +17,51 @@ import (
 const sqlBoilerVersion = "2.6.0"
 
 var (
+	cfgFile   string
```
Comment (aarondl) [inline main.go]: For global vars (usually flags) I prefer to keep them separated by their type in some way, usually I'd call this `flagConfigFile`, there's no reason to abbreviate to `cfg` either.

File: main.go [hunk 2]
```diff
@@ -71,7 +77,10 @@ func main() {
 		SilenceUsage:  true,
 	}
 
+	cobra.OnInitialize(initConfig)
+
 	// Set up the cobra root command flags
+	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file")
```
Comment (aarondl) [inline main.go]: The comment here should be more descriptive. "Use this flag to override the default config file lookup" or something along those lines

**Comment by l4u** [comment]
Thanks for reviewing it. @aarondl 
Rebased and added changes to the boil_main_test.tpl

**Comment by Dexus** [comment]
@aarondl can this be added shortly? need the function for multiple databases that used in on of our project.

Thanks, Josef

**Comment by aarondl** [comment]
Will line this up for release when I figure out what's going on with these fixes around null foreign keys in #265 and #269

**Pull request merged**